easybuild/easyconfigs/g/GPAW-setups/GPAW-setups-0.9.20000.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'GPAW-setups'
+version = '0.9.20000'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """
+This is the Atomic PAW setup data for GPAW
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://wiki.fysik.dtu.dk/gpaw-files']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = [
+    '6c71682be12a41e17909f65fd7c1a2e4a6a7becb63fbeed2f0f3a1616d6fd41f',
+]
```

Comment:
Let's make this single-line, and glued to `sources` above (no empty line in between)?